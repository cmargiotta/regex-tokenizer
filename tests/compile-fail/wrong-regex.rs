use regex_tokenizer::tokenizer;

fn main() {
    tokenizer! {
        test

        "a[1" => A
        "b" => B
        "c" => C
    }
}
