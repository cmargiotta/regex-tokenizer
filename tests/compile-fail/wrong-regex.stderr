error: Invalid regex: regex parse error:
           "a[1"
             ^
       error: unclosed character class
 --> tests/compile-fail/wrong-regex.rs:7:9
  |
7 |         "a[1" => A
  |         ^^^^^
